src/compiletest/compiletest.rs

```diff
@@ -310,11 +311,11 @@ fn collect_tests_from_dir(config: &Config,
             tests.push(make_test(config, &paths))
         } else if file_path.is_dir() {
             let relative_file_path = relative_dir_path.join(file.file_name());
-            try!(collect_tests_from_dir(config,
+            collect_tests_from_dir(config,
                                         base,
                                         &file_path,
                                         &relative_file_path,
-                                        tests));
+                                        tests)?;
```

Comment:
The automatic conversion breaks alignment in multi-line statements such as this. Should this be addressed in separate follow-up PRs?